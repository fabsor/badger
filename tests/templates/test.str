Hi {{ name }}, welcome to the testing