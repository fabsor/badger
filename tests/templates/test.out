Hi Tester, welcome to the testing